README.md

```diff
@@ -0,0 +1,34 @@
+# Sentry Anypoint Connector
+
+[Connector description including destination service or application with]
+
+# Mule supported versions
+Examples:
+Mule 3.4.x, 3.5.x
+Mule 3.4.1
+
+# [Destination service or application name] supported versions
+Example:
+Oracle E-Business Suite 12.1 and above.
+
+#Service or application supported modules
+Example:
+Oracle CRM
+Oracle Financials
+or 
+Salesforce API v.24
+Salesforce Metadata API
+
+
+# Installation 
+For beta connectors you can download the source code and build it with devkit to find it available on your local repository. Then you can add it to Studio
+
+For released connectors you can download them from the update site in Anypoint Studio. 
+Open Anypoint Studio, go to Help → Install New Software and select Anypoint Connectors Update Site where you’ll find all avaliable connectors.
+
+#Usage
+For information about usage our documentation at http://github.com/mulesoft/sentry.
+
+# Reporting Issues
+
+We use GitHub:Issues for tracking issues with this connector. You can report new issues at this link http://github.com/mulesoft/sentry/issues.
```

Comment:
@luizalfredo23 this is the default content generated for this file, right? We need to enter the proper README information for this connector. You have to provide adequate documentation if you want to make this project open source.